### PR TITLE
yup SchemaDescription test params

### PR DESCRIPTION
**Pull request by onichandame**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

This PR is trying to solve a trivial problem of yup types. An example of the problem is shown in https://codesandbox.io/s/jovial-frog-7pqym.
As shown in the example, TS complains that the 'min' property does not exist. But if you look at the console output, this property does exist. Hence a false alert.

The solution is simply redefining the type of test params.

As I know little of the codebase of yup, any suggestion that better addresses the issue is welcome.

### CODE REVIEW

**Comment by onichandame** [comment]
also added the missing type definition of a field of ObjectSchema. The example can be seen on https://codesandbox.io/s/wizardly-dewdney-hfvkz

**Review by sseppola — CHANGES_REQUESTED** [review]
Thank you for your contribution!

This all looks good to me, but for readability I would like you to change the way the tests are written, see the comment I left in the code.

I don't like to use the "Request changes" button in this case, but it's the workflow DT sets up. Please don't take it negatively, this is a good contribution.

Once fixed I'm happy to approve this unless any other maintainer has something to say.

File: types/yup/yup-tests.ts
Comment (onichandame) [reply types/yup/yup-tests.ts]: Thank you for the suggestion. please see the latest change

**Review by sseppola — APPROVED** [review]
Thanks @onichandame!

**Comment by PranavSenthilnathan** [comment]
Congratulations on your first DefinitelyTyped contribution! Thank you for being a part of the community!

**Pull request merged**
